castle/cms/browser/controlpanel/templates/status.pt

```diff
@@ -0,0 +1,59 @@
+
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      lang="en"
+      metal:use-macro="context/prefs_main_template/macros/master"
+      i18n:domain="plone">
+<body>
+<metal:main metal:fill-slot="prefs_configlet_main">
+
+    <a href=""
+       id="setup-link"
+       tal:attributes="href string:$portal_url/plone_control_panel"
+       i18n:translate="">
+        Site Setup
+    </a>
+
+
+
+    <h1> Subprocesses </h1>
+
+    <div metal:use-macro="context/global_statusmessage/macros/portal_message">
+      Portal status message
+    </div>
+
+    <div id="content-core">
+        <div tal:condition="python:view.docsplit()">
+            <span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Docsplit: OK
+        </div>
+    </div>
+    <div id="content-core">
+        <div tal:condition="python:view.docsplit() == False">
+            <span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Docsplit: ERROR <span tal:content = "structure python: view.docsplitError()"/>
+        </div>
+    </div>
+    <div id="content-core">
+        <div tal:condition="python:view.elasticsearch()">
+            <span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Redis: OK
+        </div>
+    </div>
+    <div id="content-core">
+        <div tal:condition="python:view.elasticsearch() == False">
+            <span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Redis: ERROR
+        </div>
+    </div>
+    <div id="content-core">
+        <div tal:condition="python:view.celery()">
```

Comment:
when the condition is true gives a TypeError: 'bool' object has no attribute '__getitem__'. (this is just a note for myself)

Reply:
Maybe paste the full error stack here